doc/source/containers.rst

```diff
@@ -0,0 +1,36 @@
+
+****************
+Caproto-in-a-box
+****************
+
+.. highlight:: bash
+
+.. contents::
+
+
+Build a container with Buildah
+------------------------------
+
+To build a minimal container (based of Fedora) use the following bash script ::
+
+   #! /usr/bin/bash
+   set -e
+   set -o xtrace
+
+
+   buildah images
+   buildah containers
+   container=$(buildah from fedora)
+   buildah run $container -- dnf -y install python3 ipython3 python3-pip g++ gcc
+   buildah run $container -- pip3 install ophyd databroker bluesky caproto
```

Comment:
It might be worth doing `caproto[standard]` here to get `netifaces` and `dpkt`.

Also, why include the others, especially `databroker`?

Reply:
`"minimal container"` noted above: agreed on all @danielballan's points here.

Reply:
Yeah, working on slimming it down.  The downsides of documenting the random walk.